plugins/module_utils/mysql.py

```diff
@@ -143,9 +143,25 @@ def mysql_connect(module, login_user=None, login_password=None, config_file='',
 
     if _mysql_cursor_param == 'cursor':
         # In case of PyMySQL driver:
+        if mysql_driver.version_info[0] < 1:
+            # for PyMySQL < 1.0.0, use 'db' instead of 'database' and 'passwd' instead of 'password'
+            if 'database' in config:
+                config['db'] = config['database']
+                del config['database']
+            if 'password' in config:
+                config['passwd'] = config['password']
+                del config['password']
         db_connection = mysql_driver.connect(autocommit=autocommit, **config)
     else:
         # In case of MySQLdb driver
+        if mysql_driver.version_info[0] < 2 and mysql_driver.version_info[1] < 1:
```

Comment:
```suggestion
        if mysql_driver.version_info[0] < 2 and mysql_driver.version_info[1] < 1:
```
should we use `get_connector_name` and `get_connector_version` + `LooseVersion` in the conditions? would it be more readable and safe?

Reply:
Using `get_connector_version` and `LooseVersion` would take the same data (`.version_info`), convert it to a string, parse it and compare it. I prefer the direct integer comparison as implemented in this PR. There is also a comment which should explain it.

`get_connector_name` is more intuitive than `_mysql_cursor_param`, I will use it.

Reply:
Sure thing, thanks:)